contributing.md

```diff
@@ -0,0 +1,99 @@
+# Contributing Guidelines
+
+This document will guide you through the process of contributing to our projects and help maintain a healthy and
+collaborative environment.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [Issues](#issues)
+3. [Making Changes](#making-changes)
+    - [Coding Conventions](#coding-conventions)
+4. [Pull Requests](#pull-requests)
+    - [For the Author](#for-the-author)
+    - [For the Reviewer](#for-the-reviewer)
+5. [Testing](#testing)
+6. [Questions](#questions)
+7. [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+To get started, please refer to the [bln-dev-handbook](https://github.com/bluedrop-learning-networks/bln-dev-handbook)
+and [bln-ops-handbook](https://github.com/bluedrop-learning-networks/bln-ops-handbook) for an overview of our
+development and operational processes. We highly recommend starting with the [Development Environment - From Zero to
+Tilt](https://github.com/bluedrop-learning-networks/bln-ops-plan-state/blob/master/local/README.md) article, which will
+help you set up an end-to-end local environment for our projects.
+
+Before contributing, make sure to familiarize yourself with the project's documentation, codebase, and any relevant
+issues.
+
+## Issues
+
```

Comment:
This seems like a reasonable idea, except that there is no visibility to the business to work that is not tracked in Jira. I'll see what Dave thinks about it.

Reply:
Yes, this I was about to say. So many times I have been wanting to create gh issue to track something technical. I would use them as reminders for devs.

Reply:
I had a chat with Dave about this. We can go ahead and use Github issues because it's a logical and convenient place for issues such as the examples mentioned above. We can certainly use them as reminders. However, that on its own won't result in visibility to get those issues assigned and tackled. We will either need to create them in Jira as well under "Technical backlog" or find a way to sync them to Jira.

Reply:
182ccc13e0422cd00b8ba372c7ddaa13fa47f5b5